index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>VillageFest</title>
+    <script src="https://kit.fontawesome.com/b7309bd951.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+    <section id="nav-menu">
+        <div id="cancel">&times;</div>
```

Comment:
- [ ] **REQUIRED:** As highlighted in the project requirement, our website should have a top short bar section with social media links, logout, and language links aligned to the right.
### Original design:
![image](https://user-images.githubusercontent.com/71874425/163458898-8cfd18a0-3c8d-4895-b1b7-8e933ab90dff.png)
- So, Kindly implement and add this short dark bar to our website to make it identical to the original design.